bindings/cs/MongoDB.Crypt/MongoDB.Crypt.csproj

```diff
@@ -61,6 +61,30 @@
       <IncludeAssets>runtime; build; native; contentfiles; analyzers</IncludeAssets>
     </PackageReference>
   </ItemGroup> -->
+  <ItemGroup>
+    <Content Include="$(MDBDeveloperDirectory)/mongocrypt.dll" >
+      <Pack>true</Pack>
+      <PackagePath>x64/native/windows</PackagePath>
+    </Content>
+  </ItemGroup>
+
+  <ItemGroup>
+    <Content Include="$(MDBDeveloperDirectory)/mongocrypt.pdb" Condition="'$(Configuration)'=='Debug'">
```

Comment:
Pdb files are built regardless of release vs debug mode. Does the csharp driver ship pdbs? I would match whatever it does here.

Reply:
Good catch! The C# driver does ship with PDBs in the .nupkg file, so I agree that the wrapper should do the same. I've also added the `mongocrypt.pdb` file.

Aside: Although I believe the newer way is to create a [.snupkg](https://docs.microsoft.com/en-us/nuget/create-packages/symbol-packages-snupkg) file, even the driver isn't doing this, and due to the time crunch, I'm fine with not going down this route.